### PR TITLE
Consolidate proxy to the popup home page

### PR DESCRIPTION
**Pull request by ruihildt**
This will remove the separate popup tab for the proxy and show everything in a single view.
The proxy will show basic info for all websites and the current website in separate tabs.

Advance management of the proxy will be left to the separate options page (next thing being worked on).
At some point, I'll improve the design as well (typography, spacing, etc.)

image

### CODE REVIEW

File: src/components/Buttons/SplitButton.vue [hunk 1]
```diff
@@ -31,31 +31,44 @@ const handleSubClick = () => {
...
+      :size="$props.size"
       @click="handleMainClick"
     >
       {{ props.mainText }}
     </Button>
 
     <n-popover ref="popoverRef" trigger="click" raw placement="bottom" @update:show="onUpdateShow">
       <template #trigger>
-        <Button class="h-10 flex items-center">
+        <Button :size="$props.size" :class="buttonHeightClass">
```
Comment (hankolsen) [inline src/components/Buttons/SplitButton.vue:63]: No need for `flex items-center`any longer?

**Review by hankolsen — CHANGES_REQUESTED** [review]
The Proxy tab is just empty, maybe add some kind of message there...?

image

File: src/components/Buttons/SplitButton.vue (hunk 1 above)
Comment (ruihildt) [reply src/components/Buttons/SplitButton.vue:63]: No difference at all.

**Review by hankolsen — APPROVED** [review]
Please add a `cursor: pointer` to the "the popup" link:
image

**Pull request merged**